CMakeLists.txt

```diff
@@ -566,7 +562,9 @@ endif(WIN32)
 # Extern
 add_subdirectory (extern/osg-ffmpeg-videoplayer)
 add_subdirectory (extern/oics)
-add_subdirectory (extern/osgQt)
+if (NOT ANDROID)
```

Comment:
This should be tied to USE_QT.

<pre>
if (USE_QT)
   add_subdirectory (extern/osgQt)
endif()
</pre>